makefile.shade

```diff
@@ -296,13 +296,19 @@ var buildTarget = "compile"
         }
     }
 
-#cache-coherence
+#use-local-coherence description='Sets up environment variables to use Coherence at COHERENCE_PATH'
     @{
         if (string.IsNullOrEmpty(coherencePath))
         {
             throw new Exception("COHERENCE_PATH not specified.");
         }
 
+        universeCommitsFile = Path.Combine(coherencePath, UniverseCommitsFileName);
```

Comment:
Also make sure you unset `UNIVERSE_COMMITS_FILE` on the CI configs since we want this code to build the path to it.